ups/ap_verify_ci_dc2.table

```diff
@@ -0,0 +1,2 @@
+setupRequired(obs_lsst)
+setupRequired(ap_verify)
```

Comment:
Datasets must not depend on `ap_verify`, both because that runs the risk of a circular dependency and because it's potentially confusing when you're working with a branch of `ap_verify`. However, datasets should depend on whichever PSF package was used to create the templates.